### PR TITLE
Create post_action.yaml for Retry failure strategy

### PR DESCRIPTION
**Pull request by archit-harness**
Create post_action.yaml for Retry failure strategy

### CODE REVIEW

File: schema/timeout_action.yaml [hunk 1]
```diff
@@ -11,10 +11,35 @@ properties:
...
+      - properties:
+          spec:
+            $ref: ./empty_spec.yaml
+  - allOf:
+      - properties:
+          type: pipeline-rollback
+      - properties:
+          spec:
+            $ref: ./empty_spec.yaml
 - allOf:
```
Comment (bradrydzewski) [inline schema/timeout_action.yaml:43]: check indentation

File: schema/failure.yaml [hunk 2]
```diff
@@ -36,6 +36,38 @@ properties:
...
+      - properties:
+          spec:
+            $ref: ./empty_spec.yaml
+  - allOf:
+      - properties:
+          type: pipeline-rollback
+      - properties:
+          spec:
+            $ref: ./empty_spec.yaml
 - allOf:
```
Comment (bradrydzewski) [inline schema/failure.yaml:71]: indentation

File: schema/post_action.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  type:
+    description: Type defines the failure strategy type.
+    type: string
+    enum:
+    - abort
+    - ignore
+    - success
+    - stage-rollback
+    - pipeline-rollback
+    - fail
```
Comment (bradrydzewski) [inline schema/post_action.yaml:14]: I would process `failure` for consistency

File: schema/failure.yaml (hunk 2 above)
Comment (archit-harness) [reply schema/failure.yaml:71]: done

**Pull request merged**